### PR TITLE
Update authors.yml

### PR DESCRIPTION
**Pull request by sramkrishna**
Adding the oneAPI blog. Please see https://dev.to/oneapi for samples of my blog posts. oneAPI is an industry level specifications but it's somewhat tilted towards Intel but hopefully I can convince some other industry players to write something. I mostly will speak about the technology and avoid my employer.

### CODE REVIEW

**Comment by vsoch** [comment]
Okay taking a look! The frequency looks good -  I actually only see one post generated so it wouldn't overwhelm the feed.  It looks like this was just started in October 2022 - what is your plan for frequency?

In terms of content, this particular post looks good, but I do think we have to be careful about blogs that are basically promoted by vendors. E.g., right now there are two contributors, both from Intel, and oneAPI is in fact an intel product. If this was truly your personal blog it would be an easy yes, but since it's a blog that just seems to have Intel employees for an Intel product, I'm not sure it's an easy yes. If we say yes here, then the loophole might just be "make a blog that isn't on a company site but still promotes the product." 

I am conflicted - pinging @alansill and @FCLC for second opinions! On the one hand, I want to be inclusive. On the other hand, vendors and their products have not just a strong but _the_ strongest voice in the HPC community, and the goal of the blog here was to promote individual voices. Do you have a personal blog where you share this and more of your ideas?

**Comment by sramkrishna** [comment]
So a couple things.

yes, this is why I wanted to be transparent about the purpose of the blog. Especially as an Intel person. But let me see if this additional information will help you make a decision.

1) Intel doesn't own "oneAPI" - in fact the effort here is to create an industry standard around SYCL. Intel is moving to disassociate itself away from "ownership" and more towards being a consumer of oneAPI like anyone else. There is an open governance model as one would expect.

2) I'm not interested in promoting Intel. That would defeat the purpose of disassociating it from oneAPI. In my role, I see myself as promoting the industry not Intel. Thus, for instance - in my meetup.com I'm inviting Aksel to talk about hipSYCL. Why? Because people should know, the community should know that being part of oneAPI means that there are many players, including that has no connections to corporations.

In the same vein, my blog posts are going to focus on code samples, build systems, alternative ways to write SYCL code - conversations about ethics. Things that illume. I'm not interested in commercializing our public space. :-) In this space, it's important that my reputation is clean to accomplish my goals. Building communities is almost always about trust. 

The only time I might include Intel is if we are doing oneAPI release - that information more often than not will come from Intel. Hopefully, that will come from the open governance forum.

**Comment by vsoch** [comment]
Totally understand! It's hard to say where to draw the line, which is why I want feedback from @alansill and @FCLC. E.g., my vision of these blogs is not for projects, but the _people_, meaning personal blogs. If someone writes about projects? Great! But a blog explicitly for a single project I think is toe-ing the gray area line - e.g.,, even though these are not your intentions, it provides a loophole for any vendor to create a project blog, present it as such, and then take the focus away from the individuals and back to a product. 

But that said, I think maybe we should have a different place or strategy for exactly these kind of projects! With some kind of separation then a reader can knowingly browse to either source and there is transparency about the content. Looking forward to hearing thoughts from @FCLC and @alansill

**Comment by sramkrishna** [comment]
Fair enough :-) It's about people - the problem here is that if we were take out the oneapi bit, my passions aren't always on HPC topics - it's on stuff like linux desktop, DEI, etc.  But yeah, also you know, we can do a trial run and if it doesn't work out there will be no hard feelings. I'm a get along kind of person. :)

**Comment by sramkrishna** [comment]
I didn't answer your question on frequency - I think I'll likely blog about
2x a month. Not more than that. Writing this stuff takes a lot of time!

**Comment by FCLC** [comment]
~~Catching up, will edit this in a moment with thoughts;~~


_**TL;DR:**_ For a plethora of reasons, I'm OK with this going in, but think any vendor adjoint blogs do need to be put under a stricter eye prior to being added, and there needs to be an understanding that there's no mandate to host/share the content. 

I'm having a slight chuckle. We were already questioning how to deal with this sort of eventuality (intersection of vendors, dev-rel, marketing etc.)

See : https://github.com/hpc-social/blog/pull/12#issuecomment-1332780007

From a *personal* standpoint I trust and take @sramkrishna at his word. But adding a vendor affiliated blog, even if the intention is for that vendor to disassociate (and Intel has been making serious strides to do so) still feels a little off putting. 

I see the HPC.social blog as: Cool people talk about cool things without trying to sell it to anybody. See the Xilinx example in the CaRCC PR above. 

Speaking to OneAPI, Intel has handed over stewardship to Codeplay. They do *own* Codeplay, but Codeplay is its own entity. They have also been working actively to make OneAPI vendor agnostic, and have been actively putting resources behind supporting implementations that "compete" against their own.
See promoting HipSYCL, having AMD Zen specific optimizations in Intel MKL, having Intel Clear Linux be the best performing AMD Linux distro etc. (Side bar: IMO they learned from the fuster cluck of the mathworks debacle a few years back and changed for the better). 

They've been pushing SYCL as the open Kronos spec and of the "big" vendors have probably been the most FOSS friendly in recent years. I'm more willing to take a wait and see approach with an Intel affiliated project than a vendor that actively antagonizes HPC/FOSS users for example.

There should also be an understanding that if you are going to be sharing your content here, that the content isn't a sale blurb/pitch (regardless of if you're an individual or vendor affiliated) 

I'd also like to think that the blog, even if primarily HPC, isn't only HPC. 

It's more about resource sharing by people in and around HPC for people in and around HPC. 

See for example this post about learning new langs from Adam: https://hpc.social/blog/2022/toy-programs-for-learning-a-new-language/

As for the scope/single project'ness of OneAPI, I don't know that we need to worry about it being too small.

The core concept is to be the actual "do all the things *on* all the things" API. partially why it uses SYCL, but other parts of it like OneDNN, IntelMKL, the FPGA libraries, the multimedia libraries, primitives/support for Windows, Linux, some of the BSDs, some of MacOS etc. 

You could write actual books on just what OneAPI currently does, without talking about what it's still trying to do. 

Overall? I'm fine with it, trust @sramkrishna, and think that if/when the directions of the blog (either HPC Social or OneAPI blog) we're all going to be fine with reading back/pointing to this same issue and removing the auto updater. 

<details>
  <summary markdown="span">Disclosures</summary>
I've interviewed at Codeplay and am in active talks with Intel. Same token, I'm scheduling an interview with LLNL with Vscoch's team while I type this. 
</details>

**Comment by vsoch** [comment]
I think we can find a way to be inclusive - the question is just where to put it! We want you to be a part of the community blogspace @sramkrishna.

**Comment by sramkrishna** [comment]
I'm happy to be part of however you think I would make the most impact - I don't need to be in the blogs if it runs counter to what you're trying to accomplish as community leaders :-) I'm sure it will naturally shake out ! 👍

**Comment by FCLC** [comment]
> Adding the oneAPI blog. Please see https://dev.to/oneapi for samples of my blog posts. oneAPI is an industry level specifications but it's somewhat tilted towards Intel but hopefully I can convince some other industry players to write something. I mostly will speak about the technology and avoid my employer.

Added my thoughts, concerns etc. above in a way we can refer back to if need be years down the line for the OneAPI or other vendor adjoint blogs. Because I forget if GH sends pings on edits that include tags, sending a ~~bat~~ dino signal to @vsoch

**Comment by vsoch** [comment]
@FCLC what about if we had a separation of spaces for projects/vendor blogs and personal blogs? I think I also trust @sramkrishna but it sets a precedence that (in the future) might force us to act unfairly or in a way that (from an outside standpoitn) it doesn't make sense. We need a strong policy that determines what belongs here (vs. something else) and it's going to get really hard if we muddle that.

**Comment by FCLC** [comment]
> @FCLC what about if we had a separation of spaces for projects/vendor blogs and personal blogs? I think I also trust @sramkrishna but it sets a precedence that (in the future) might force us to act unfairly or in a way that (from an outside standpoitn) it doesn't make sense. We need a strong policy that determines what belongs here (vs. something else) and it's going to get really hard if we muddle that.

Agreed. Perhaps using a tag for any vendor/vendor affiliated/adjoint blogs? Then when that vendor tag is present, have is display under the title, the same way we have a cross post blurb now? 
It's a little heavy handed but also also relatively clean, super upfront etc. (thinking kind of like when say on YT it says "this publication is affiliated with XYZ government"). Seems like a form of responsible disclosure. 

(Should we technically be asking people to disclose if they're currently at a vendor for that matter? even on a personal blog? 
Content that contains sponsored hardware vs being explicitly sponsored is a different story, but I don't know that we should discuss that in this issue. Nor do I have the BW to deal with that tonight 😅 Convo for a different day.)

In terms of a policy, not sure. 

Something like a draconian 1 strike you're out seems a bit much, but having a seat of the pants:
"We reserve the right to remove any post that the 'Executive Committee' deems by simple majority to have been overtly commercial in content in nature."

There's a larger question regardless of if it's a vendor or not that's along the lines of: Who decided to give us that control? Why should we be empowered to do so? Sure an Exec Committee expands the responsibility to more people, but who chooses those people? Do we need an editorial board? 

Because this is a small friends/colleagues type thing I don't *think* it's a problem to have a primarily faith based, "don't be stupid pls 🥺"  approach. 


We also don't want a system that needs every post to be approved either, else it's a pain to scale. 

So I'd propose something along the lines of: 

All vendors affiliated/adjoint blogs must provide us with a person to contact. If any of the active moderators/administrators receive a complaint or themselves feel that a piece of content crosses the line between "Here's a cool thing we're doing" and "Here's a cool thing that you should be doing" or any other sort of breach of trust, it will be the responsibility of that vendor contact to rectify the issue/provide reasonable context/alternatives/updates or have that specific post taken down. Should multiple breaches of trust occur, that vendor will be delisted from the HPC.social syndicated blog.

**Comment by vsoch** [comment]
> All vendors affiliated/adjoint blogs must provide us with a person to contact. If any of the active moderators/administrators receive a complaint or themselves feel that a piece of content crosses the line between "Here's a cool thing we're doing" and "Here's a cool thing that you should be doing" or any other sort of breach of trust, it will be the responsibility of that vendor contact to rectify the issue/provide reasonable context/alternatives/updates or have that specific post taken down. Should multiple breaches of trust occur, that vendor will be delisted from the HPC.social syndicated blog.

That works for me. If @alansill gives the A-OK do you want to PR this policy to the about page? I think if this happens, since the posts are retrieved nightly we would need to add an ability to tell the updater "don't add posts before this date" otherwise the offending / removed post would just be added again. Should we also say something about frequency? E.g., a vendor blog that posts once a week is going to drown out everything else. I also think we should have a filter for blog types, of course that would first require categories (e.g., personal, project, vendor maybe?) 

@FCLC I have a lot on my plate at the moment so if you want to jump into this work please go ahead!

**Comment by sramkrishna** [comment]
>  @FCLC <https://github.com/FCLC> what about if we had a separation of
spaces for projects/vendor blogs and personal blogs? I think I also trust
@sramkrishna <https://github.com/sramkrishna> but it sets a precedence that
(in the future) might force us to act unfairly or in a way that (from an
outside standpoitn) it doesn't make sense. We need a strong policy that
determines what belongs here (vs. something else) and it's going to get
really hard if we muddle that.

I agree. It sets a bad precedent. I think it's worth just having personal
blogs and leave corporate attached personal blogs like mine out of it for
now. This way we defer the decision making and think about the right
policy. So here is what I think:

* Current policy is personal blogs at this time for a period of 6 months.
* We will revisit based on what the needs of the community are and how
mature the community has grown.

Corporate associated blogs like mine can be advertised on mast.hpc.social
and boosted by community members for viewership. This gives people personal
power to boost as they see fit.

If you want to take an example of aggregate blog, the primary community I
come from is GNOME and we have planet.gnome.org which is a list of
aggregate blogs. Here is the policy from https://wiki.gnome.org/PlanetGnome:

*What is a good blog for Planet GNOME?*

While there is no clear-cut definition of what is a good blog for Planet
GNOME, it certainly helps to keep in mind that we want Planet GNOME to be
most interesting to the readers. This is why we have editors. However,
there are a few things that you can easily check to make sure that your
blog would make a good addition:

   - the target audience of your posts is not restricted to your family or
   close friends only. We want readers of Planet GNOME to read and care about
   most of your posts.
   - some posts should be relevant to the GNOME community, either because
   they're related to GNOME, some underlying projects (like freedesktop.org
   projects), some technologies using GNOME, etc. or because it's a topic most
   people in our community care about, like freedom. Keep in mind that we
   cannot list all topics that are of interest to the GNOME community here.
   - the posts are English only. The point is that if you mix Chinese,
   Dutch, Farsi, French, German, Italian, Spanish, it will look nice, but it
   will be a nightmare for the readers. There are other Planets for various
   languages: having multiple Planets is how we want to encourage the
   diversity of languages in the GNOME community -- without forcing people to
   have to skip posts in languages they cannot understand. We can eventually
   help you setup a planet for your language if needed.

**Comment by alansill** [comment]
I haven't followed all the discussion on this topic so will have to read back a bit and catch up. The issues I see are that the current Fediverse community seems by and large very resistant to the idea of anything that looks like advertising; but exactly in contraposition to this is that they very much want to stay up to date with the best hardware and software options in the field, and by extension, want news about these topics. I pointed this out to Patrick Kennedy, who seems to have become too busy to engage with the transitions in the community; but to be fair I think this is an unsolved topic in the newly emerging community here. 

See the results of the survey here. We may have to conduct a new survey to get the answer. 

https://survey.hpc.social/results/hpc-social-design-survey.html
image

**Comment by vsoch** [comment]
@alansill I think hpc.social as a larger community can support all kinds of blogs and content, but this community _member_ blog was created (by me) with the individuals. people in mind. If the larger community wants this content we should provide it, but the voice of this other kind of content should not be allowed to overpower the individuals.

**Comment by sramkrishna** [comment]
> @alansill <https://github.com/alansill> I think hpc.social as a larger
community can support all kinds of blogs and content, but this community
*member* blog was created (by me) with the individuals. people in mind. If
the larger community wants this content we should provide it, but the voice
of this other kind of content should not be allowed to overpower the
individuals.

I'm in agreement - so, let's respect that. My proposal is to see how things
evolve - if there is a space for my kind of content then we can find it.
HPC is an extension of my work so I don't have personal stuff to say unless
it involves sustainability (eg datacenters) - the software part involves my
employer. At this moment of time, it's still dominated by them.. Also, the
somewhat more interesting content like benchmarks and the like I am
personally constrained by my employer to do without a lot of red tape. I'm
ok with just advertising my blog content on mastodon and it will still be
curated from specifically talking about my employer but focused on the
software.

**Comment by alansill** [comment]
With all of the above in mind, I think there's enough interest in OneAPI as a technology to allow the blog. The survey did show there is interest it this type of content so I think we should go ahead and be prepared to adjust and clarify content guidelines based on the response.

**Comment by vsoch** [comment]
Okay here is a proposal. Akin to how we have /blog, we can create two more repos:

- community-project
- community-news

The first will primarily be for blogs like @sramkrishna - a clearly branded blog for a specific project, and not a personal blog but also not a vendor blog. The second will be more akin to what we see on twitter feeds - any sources that provide updates about hardware, news - and actually since I know most of these resources I can likely add a lot of them to get the space started.

The trick here will be having them be distinguished based on these categories, but also maybe connected to one another so it's easy to move back and forth. I'm thinking in this top level space (maybe centered) there can be three buttons that link to the other blogs:

image

And the reason is keep them on separate sites is really just spreading out the automation and organization of things. What do you think?

**Comment by vsoch** [comment]
@alansill we are talking around one another - I agree there is interest in OneAPI and we should include it somewhere in the hpc.social community, but I disagree that it should be added to the blog here, which is for individual personal blogs.

**Comment by alansill** [comment]
Hmm.

**Comment by alansill** [comment]
How about a survey or small poll on Mastodon?

**Comment by vsoch** [comment]
I would be very upset if you allowed for a vote that then bull-dozered my entire effort to make a community blog space. @sramkrishna is in agreement with me and I think we need to just figure out another blog space to put this content.

**Comment by vsoch** [comment]
@alansill could you please walk me through the concerns you have so I can understand? I'm not sure if you don't understand me or disagree with me.

**Comment by sramkrishna** [comment]
I interpreted @alansill's comment to have a poll deciding whether they want
content like mine and not necessarily overriding your vision for the blogs.
Just a method to gain clarity.

Also, none of you should worry that saying no means it is a rejection. I
love this community and am happy to keep investing in it. I've already
gotten some of my folks at my employer to invest their time here. The more
the merrier and that oneAPI content is going to come out naturally on
mastodon. So feel free to take your time on where you want this kind of
content.

:)

**Comment by vsoch** [comment]
> Also, none of you should worry that saying no means it is a rejection. I
love this community and am happy to keep investing in it. I've already
gotten some of my folks at my employer to invest their time here. The more
the merrier and that oneAPI content is going to come out naturally on
mastodon. So feel free to take your time on where you want this kind of
content.

Oh I don't want to say no! I want to start a special place just to say yes. <3 

Having separate portals with clear policy (and no gray area) about criteria for adding content is ideal I think.

**Comment by sramkrishna** [comment]
OK - so I like your current proposal: Personal blogs aggregator - the
"soul" of the HPC community - HPCers who are personally invested in the
minutiae of the work they are doing, the projects they are working on with
some content about their culture and pet pictures :D - things that we all
find in common and share and talk about.

Community content aggregator - This is content from people who represent
projects, ecosystems, or governance boards that talk about specific
community interested content around the work they represent. The content
range from release notes, tricks and tips, discussion around tooling and
infrastructure, and other things that are neutrally branded. Discussion of
branded topics like CUDA, SYCL, and oneAPI are ok - discussions about
hardware are ok. Product announcements are not ok especially.

**Comment by alansill** [comment]
I donʻt think a poll (Elonʻs experience to the contrary) is intented to bulldoze anyoneʻs opinion. Iʻm just curious whether people are sensitive at all on this issue. We can still choose to separate them but I want to be clear with the community that we may add blog feeds, together or separate, from groups and projects including vendorbased open source projects. Itʻs worth noting that in the pre-launch survey, getting info on opportunities to joing projects wasnʻt at all high on peopleʻs lists so we may be exploring the wrong end of the problem here. What people semed to want was to be kept up to date on the best available hardware and software. So I donʻt have a strong opinion here to push.

I keep trying to get out of the way of going forward on this topic and somehow I keep failing to do so!

**Comment by FCLC** [comment]
Concern is: 
Vendors on current blog undermines it as a community blog

Proposed solutions: new/adjacent blog for vendor/vendor related content. 

Proposed implementation: 

New Project and news content categories that do allow for vendor affiliated content, that are explicitly separate from community/personal content. 
 
Project feed becomes home for projects that vendors, and other orgs support. Say for example Spack wanted to share posts on their progress. It's very much HPC, but it's also more of an organization. In that case it would probably be better categorized in the "Projects" feed. 

The News feed becomes a sort of home for announcement/press releases.

So we have the: 
1. HPC Community blog, focused on HPC people and what People are doing. 

2. HPC affiliated projects, which can include cross posts from personal projects, but also Vendor projects, National Lab projects, academia projects etc. 

3. HPC news feed. A sort of HPC only, automated version of a news feed. (Perhaps it becomes a sort of Techmeme for HPC?) 


1 We have a system in place 


2 We have a rough policy/wording that V are fine with, @alansill are you ok with it?

@sramkrishna putting your "vendor" cap on, do you see anything in that text ripe for abuse? Or the opposite, that would make a vendor pause or not want to partake?

text is: 

> All vendors affiliated/adjoint blogs must provide us with a person to contact. If any of the active moderators/administrators receive a complaint or themselves feel that a piece of content crosses the line between "Here's a cool thing we're doing" and "Here's a cool thing that you should be doing" or any other sort of breach of trust, it will be the responsibility of that vendor contact to rectify the issue/provide reasonable context/alternatives/updates or have that specific post taken down. Should multiple breaches of trust occur, that vendor will be delisted from the HPC.social syndicated blog.

We'd modify it to be in line with the new project section delineation, but the core idea would stay the same. 

3  Would probably be done by watching an RSS feed of the major sites that post HPC news (STH, Next Platform, Register, Phoronix, HPC wire) etc.


Example cases:
If Todd G. (of LLNL/SPACK) wanted to talk about what goes into packaging software binaries on a personal blog, I'd see that as content appropriate for either community or projects. Posting to both would probably be fine. 

But if that post is on/for the official SPACK blog/LLNL blog, I'd consider that as only being appropriate for Projects. 

Speaking to the point of community member interest, survey did show that most want to be kept in the know of what is going on, but since they don't want to be advertised to, it implies a want for a sort of neutral voice/third party acting as an arbiter/ increasing the signal to noise as it were (*cough* hpc_guru *cough cough*) 

My main point here is that users want to know there's a place they can find all the latest press releases, papers etc. even if it's in a headline only list format, and I think the news section can do that. 

If they want a third party to cut through/ find the juicy news in the deluge of announcements, the guru amongst other active community members on the various platforms (mastodon, twitter etc.) act as that filter. 


Does that sound workable/good? 


The actionable items would be: 
Creating the sections 
Adding all 3 to the main site
Setting up/adding the auto tag responsible disclosure flair 
Adding a reporting mechanism (google form?) for concerned users 
Setup RSS feed scrapers for the news sections
Create, flesh out then Add content policy appropriate to each section

Nice to have: HPC social newsbot: mastodon bot that shares any articles/posts/etc from each of the sections? 

I think that's most of it?

I'm going to crash for the night, LMK what I've missed. Probably some actionables, some nuances in community and a few other things. 


P.S. @vsoch for the PR, is it mainly the about page thing you want me to add, or design the categories with attached/auto responsible disclosure based on vendor/org type? if it's the categories, happy to help/do it, but I'm really unfamiliar with the code base so will be slow ramp.

**Comment by alansill** [comment]
I think this is shaping up nicely in terms of hte first two parts, so will concentrate on the latter. I think it would be *terrific* if we could come up with a way for HPC journalists and trade publications to have their content linked or aggregated on our site somehow, optionally inside or outside of our Mastodon instance. Finding a way to do so that would leverage the Fediverse but (and) allow people to choose whether to see this content would be fantastic.

**Comment by sramkrishna** [comment]
I would also add that by submitting a PR to have your blog added to the
blogroll/rss feed that you are agreeing to abide by the HPC Social Code of
Conduct. That way we're covering the blog content of both personal and
project.

Otherwise, this seems fine. We'll need to define the tags right?

**Comment by vsoch** [comment]
:raised_hands: thank you @FCLC (and @sramkrishna ) I feel like you are hearing me spot on. This is exactly the proposal that I was trying to communicate earlier. The list of "next steps" depends on how we want the design to be. For example:

1. For separate spaces to be unified in a single root, we create a "blogs" portal of sorts that has three giant links to each of personal, news, and projects/community, and this is also where we put the about sections and the criteria for each. We could also serve "one feed to rule them all" from here that (nightly) aggregates the individual feeds (this is the approach I like). @alansill this would solve the problem of "I just want one feed" without requiring people to subscribe to all content types.
2. For separate places to exist alongside one another, we likely put the criteria in the GitHub org README and then have each blog link to the others, nothing more complex than that.

The other stuff is just details - I don't normally make sure a detailed TODO but I can if you want to work on it together!

**Comment by sramkrishna** [comment]
I'm not so sure about HPC Journalists - but I think trade publications are
good. I think we'll need to vet journalists a bit better mostly because I
don't want them to use this site as an additional way to get clicks. For
instance, I would have a problem with Phoronix being aggregated here.

**Comment by vsoch** [comment]
Starting on this soon!

**Comment by vsoch** [comment]
okay - I've finished the work, and welcome to the hpc.social community blog @sramkrishna ! Details here https://mastodon.social/@vsoch/109571939806721310 and you can see your oneapi feed here https://hpc.social/community-blog/. 

I think we will likely have more discussion for how to further improve the syndication (e.g., I didn't have time tonight to intelligently combine all those feeds into one place and chose instead to forward along one of the feeds) but hopefully we can do this in the future. Closing here, for further discussion let's do slack, or an issue/discussion around the repos:

- https://github.com/hpc-social/blog (here)
- https://github.com/hpc-social/personal-blog
- https://github.com/hpc-social/community-blog

I also linked the discussion here in the various readmes so it will be findable in the future. And @FCLC @sramkrishna I might nto have gotten in all of your good suggestions and points, so please PR and make tweaks to the sites! One quick comment - if we find we are doing changes to the actual template of the blogs, then we should make a shared template that both sites use. I'll keep an eye out for this - I'm OK as it is now as long as I don't have to update templates in two places in the future. :laughing: 

Closing now - should probably eat some dinner! Thanks for the good conversation!

**Comment by sramkrishna** [comment]
oh wow!  thank you so much! Glad that I can be a part of this community - I've already queued up a blog post that I will be posting today :) (and no vendor mentions!)
